### PR TITLE
Andrew's polish

### PR DESCRIPTION
**Pull request by Andrewdt97**
Mostly formatting (line breaks, line length, removing commented out code, etc) but with the following functional changes:

Add My Matches button to adopt pet (all code was there, just missing XML)
Fixed like button advancing dogs properly
Added login fail message
Removed Log statements
Added Other breed to breed list

### CODE REVIEW

File: app/src/main/java/edu/calvin/cs262/teamc/MatchInfo.java [hunk 1]
```diff
@@ -1,5 +1,11 @@
 package edu.calvin.cs262.teamc;
 
+/** MatchInfo
+ *
+ * Contains information on matches
+ *
+ * @author Andrw Thomas
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/MatchInfo.java]: You spelled your own name wrong, lol. I'm guessing this was not intentional.

File: app/src/main/java/edu/calvin/cs262/teamc/MyMatchesActivity.java [hunk 2]
```diff
@@ -51,35 +59,34 @@ protected void onCreate(Bundle savedInstanceState) {
...
                         JSONObject object = arrJson.getJSONObject(i);
-                        Log.e("GetDogs-Name",object.getString("dogName"));
                         String Name =object.getString("dogName");
-                        Log.e("GetDogs-Breed",object.getString("dogBreed"));
                         String Breed =object.getString("dogBreed");
-                        Log.e("GetDogs-ProfilePhoto",object.getString("profilePicture"));
                         String Photo =object.getString("profilePicture");
 
                         MatchInfo mi = new MatchInfo(Name,Breed,Photo);
                         matches.add(mi);
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/MyMatchesActivity.java:83]: These few empty lines could maybe be deleted too.

File: app/src/main/java/edu/calvin/cs262/teamc/MyMatchesActivity.java [hunk 3]
```diff
@@ -97,39 +104,41 @@ public void onResponse(String response) {
...
                 Intent i = new Intent(getBaseContext(), MatchActivity.class);
+
                 i.putExtra("loginID",getIntent().getStringExtra("loginID"));
                 i.putExtra("name", matches.get(position).getName());
-                Log.e("imgID", matches.get(position).imgSrc);
                 i.putExtra("imgId", matches.get(position).imgSrc);
+
                 startActivity(i);
             }
         });
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/MyMatchesActivity.java:132]: Extra line

File: app/src/main/java/edu/calvin/cs262/teamc/MyMatchesActivity.java [hunk 4]
```diff
@@ -149,28 +158,22 @@ public View getView(int position, View convertView, ViewGroup parent) {
...
             TextView dogNameTv = (TextView) rowView.findViewById(R.id.dogNameTv);
             TextView dogBreedTv = (TextView) rowView.findViewById(R.id.dogBreedTv);
-            Log.e("getView-POsition", String.valueOf(position));
-            Log.e("getView-Values",String.valueOf(values.get(position).getImgSrc()));
-
-            Log.e("Size", String.valueOf(values.size()));
-
 
             dogNameTv.setText(values.get(position).getName());
             dogBreedTv.setText(values.get(position).getBreed());
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/MyMatchesActivity.java]: You have two empty lines here, maybe delete one?

File: app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java [hunk 5]
```diff
@@ -99,23 +103,23 @@ public void onResponse(String response) {
...
 
                     String encodedString = object.getString("profilePhoto");
-                    encodedString = encodedString.replace("data:image/jpeg;base64,","");
+                    encodedString = encodedString
+                            .replace("data:image/jpeg;base64,","");
                     byte[] imageBytes = Base64.decode(encodedString.getBytes(), 0);
-                     currentDefault = BitmapFactory.decodeByteArray(imageBytes,0,imageBytes.length);
+                     currentDefault = BitmapFactory.decodeByteArray(
+                             imageBytes,0,imageBytes.length);
                     image.setImageBitmap(currentDefault);
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java]: Might as well delete this empty line too

File: app/src/main/java/edu/calvin/cs262/teamc/activity_login.java [hunk 6]
```diff
@@ -115,31 +120,42 @@ public void logIn(View view) {
...
                     if (responses.equals("valid"))
                     {
                         logIn.putExtra("loginID",usernameL.getText().toString());
                         startActivity(logIn);
                         finish();
                     }
+                    else {
+                        Toast.makeText(activity_login.this,
+                                "Invalid username and password", Toast.LENGTH_SHORT).show();
+                    }
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/activity_login.java:152]: Delete line (152) ?

File: app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java [hunk 7]
```diff
@@ -239,7 +225,7 @@ public void onMyMatchesBtnPressed(View view) {
      */
     private class ImageAdapter extends PagerAdapter {
         Context context;
-        List<Dogs> dogs = new ArrayList<Dogs>();
+        List<Dogs> dogs;
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java:225]: Delete line (229)?

File: app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java [hunk 8]
```diff
@@ -129,19 +129,17 @@ public void onErrorResponse(VolleyError error) {
             protected Map<String, String> getParams() throws AuthFailureError {
                 Map<String, String> postMap = new HashMap<>();
                 postMap.put("Content-Type","application/json");
-                //..... Add as many key value pairs in the map as necessary for your request
                 return postMap;
             }
         };
-        //make the request to your server as indicated in your request url
         Volley.newRequestQueue(getApplicationContext()).add(stringRequest);
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java]: Delete line (136)?

File: app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java [hunk 9]
```diff
@@ -111,16 +114,13 @@ public void onPageSelected(int position) {
...
-                    Log.e("onResponse-Length", String.valueOf(dogs.size()));
-
                 } catch (JSONException e) {
                     e.printStackTrace();
                 }
             }
         }, new Response.ErrorListener() {
             @Override
             public void onErrorResponse(VolleyError error) {
-               // Toast.makeText(activity_adoptPet.this, "Failed", Toast.LENGTH_SHORT).show();
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java]: Delete line (124)?

File: app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java [hunk 10]
```diff
@@ -59,25 +59,24 @@ protected void onCreate(Bundle savedInstanceState) {
...
+                        String photo = object.getString("photo");
                         Log.e("GetDogs-Breed",object.getString("name"));
-                        String name =object.getString("name");
+                        String name = object.getString("name");
                         Log.e("GetDogs-ProfilePhoto",object.getString("breed"));
-                        String breed =object.getString("breed");
+                        String breed = object.getString("breed");
                         Log.e("GetDogs-ProfilePhoto",object.getString("gender"));
-                        String gender =object.getString("gender");
+                        String gender = object.getString("gender");
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java]: Delete line (81)?

File: app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java [hunk 11]
```diff
@@ -59,25 +59,24 @@ protected void onCreate(Bundle savedInstanceState) {
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java]: Delete line (60)?

File: app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java [hunk 12]
```diff
@@ -232,7 +213,6 @@ public void onErrorResponse(VolleyError error) {
                 //make the request to your server as indicated in your request url
                 Volley.newRequestQueue(getApplicationContext()).add(stringRequest);
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java]: Delete line (215)?

File: app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java [hunk 13]
```diff
@@ -200,7 +182,6 @@ else if (ImageURL.getText().toString().equals(""))
             }
             else {
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java]: Delete line (184)?

File: app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java [hunk 14]
```diff
@@ -168,28 +156,22 @@ public void saveNewPetSwipe (View view) {
         TextView ImageURL = findViewById(R.id.chooseFile);
 
         if (TextUtils.isEmpty(NameText.getText())) {
+
             NameText.setError(getString(R.string.Name_Required));
+
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java:156]: Why these extra lines here (159, 161, 163)?

File: app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java [hunk 15]
```diff
@@ -136,25 +129,20 @@ public void fileSelected(final File file) {
             }
         });
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java]: Delete line (131)?

File: app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java [hunk 16]
```diff
@@ -118,9 +114,6 @@ public void onClick(View v) {
                     public void fileSelected(final File file) {
 // Here, thisActivity is the current activity
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java]: This comment is not well-aligned with the text

File: app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java [hunk 17]
```diff
@@ -261,9 +264,5 @@ public void onErrorResponse(VolleyError error) {
         //make the request to your server as indicated in your request url
         Volley.newRequestQueue(getApplicationContext()).add(stringRequest);
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java]: Delete this line too (266)?

File: app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java [hunk 18]
```diff
@@ -230,18 +235,16 @@ public void saveChanges(View view) {
         final String imgStringnew = imgString.substring(imgString.indexOf(",")+1);
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java]: Delete line (237)?

File: app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java [hunk 19]
```diff
@@ -177,16 +183,14 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
...
         super.onRequestPermissionsResult(requestCode, permissions, grantResults);
         if (requestCode == MY_CAMERA_PERMISSION_CODE) {
             if (grantResults[0] == PackageManager.PERMISSION_GRANTED) {
-                Toast.makeText(this, "camera permission granted", Toast.LENGTH_LONG).show();
                 Intent cameraIntent = new
                         Intent(android.provider.MediaStore.ACTION_IMAGE_CAPTURE);
                 startActivityForResult(cameraIntent, CAMERA_REQUEST);
-            } else {
-                Toast.makeText(this, "camera permission denied", Toast.LENGTH_LONG).show();
             }
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java]: Delete line (195)?

File: app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java [hunk 20]
```diff
@@ -177,16 +183,14 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java]: Delete line (184)?

File: app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java [hunk 21]
```diff
@@ -99,23 +103,23 @@ public void onResponse(String response) {
...
+                            .replace("data:image/jpeg;base64,","");
                     byte[] imageBytes = Base64.decode(encodedString.getBytes(), 0);
-                     currentDefault = BitmapFactory.decodeByteArray(imageBytes,0,imageBytes.length);
+                     currentDefault = BitmapFactory.decodeByteArray(
+                             imageBytes,0,imageBytes.length);
                     image.setImageBitmap(currentDefault);
 
-
-                       }
+                     }
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java]: Delete line (114)?

**Review by carolinecarlson — COMMENTED** [review]
Sorry for the crazy number of comments. They are mostly about deleting some empty lines of code. I get having one extra line of code to separate blocks of code, but I've always thought that one line was enough separation. I've commented where you use two or more empty lines, and I'm probably missing more instances where I couldn't comment. This could be just a difference in our two styles. I would like to be consistent if we can, (though empty lines of code is the least of our worries)!

There are two comments that do not deal with deleting lines, so make sure you see those.

File: app/src/main/java/edu/calvin/cs262/teamc/MatchInfo.java (hunk 1 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/MatchInfo.java]: bahahahahahaha

File: app/src/main/java/edu/calvin/cs262/teamc/MyMatchesActivity.java (hunk 3 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/MyMatchesActivity.java:132]: I assume you mean 129, which is supposed to be there.

File: app/src/main/java/edu/calvin/cs262/teamc/MyMatchesActivity.java (hunk 4 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/MyMatchesActivity.java]: Good catch. To have one empty line is very helpful, to have "two looks like carelessness."

File: app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java (hunk 5 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java]: Deleted

File: app/src/main/java/edu/calvin/cs262/teamc/activity_login.java (hunk 6 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/activity_login.java:152]: I kinda like it to make the if-else stand out more? Idk, what are your thoughts?

File: app/src/main/java/edu/calvin/cs262/teamc/MyMatchesActivity.java (hunk 2 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/MyMatchesActivity.java:83]: Deleted

File: app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java (hunk 7 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java:225]: Deleted

File: app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java (hunk 8 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java]: Deleted

File: app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java (hunk 9 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java]: Deleted

File: app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java (hunk 10 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java]: Deleted

File: app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java (hunk 11 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java]: Deleted

File: app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java (hunk 12 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java]: Deleted

File: app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java (hunk 13 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java]: Deleted

File: app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java (hunk 14 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java:156]: Not sure exactly which lines you are referring to (cause I have deleted so many lines in the file so far), but I just checked around here and I'm not seeing any major problems.

File: app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java (hunk 15 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java]: Deleted

File: app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java (hunk 16 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java]: Alligned

File: app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java (hunk 17 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java]: Deleted

File: app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java (hunk 18 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java]: Deleted

File: app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java (hunk 19 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java]: Deleted

File: app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java (hunk 21 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java]: Deleted

File: app/src/main/java/edu/calvin/cs262/teamc/activity_login.java (hunk 6 above)
Comment (carolinecarlson) [reply app/src/main/java/edu/calvin/cs262/teamc/activity_login.java:152]: That's fine. I think having just a bracket on a line (ex: line 151) has the same effect.

**Review by carolinecarlson — APPROVED** [review]
Looks good; thanks for deleting the empty lines.

**Pull request merged**